tests/open_alchemy/helpers/test_peek/test_peek_extension.py

```diff
@@ -134,23 +175,32 @@ def test_primary_key(schema, expected_primary_key):
     assert returned_primary_key == expected_primary_key
 
 
+@pytest.mark.parametrize(
+    "schema",
+    [
+        ({"x-tablename": True}),
+        pytest.param({"x-open-alchemy-tablename": True}, marks=pytest.mark.xfail),
```

Comment:
I ran this and it didn't fail for me?

Reply:
You must run `pytest --runxfail`, other these tests are skipped because they are expected to fail.